metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -1579,9 +1581,51 @@ class MetalsLspService(
     indexer.indexWorkspaceSources(buildTargets.allWritableData)
   }
 
-  def shutDownBloop(): Boolean = bloopServers.shutdownServer()
-
-  def isBloop(): Boolean = bspSession.exists(_.main.isBloop)
+  /**
+   * We only support restarting bloop and sbt servers.
+   */
+  def restartBspServer(): Future[Boolean] = {
+    def emitMessage(msg: String, msgType: MessageType = MessageType.Info) = {
```

Comment:
Doesn't `scribe.info` do the same?

Reply:
I'm not logging. I'm showing the info to the user.

Reply:
Shouldn't it be `showMessage` ?

Reply:
Yeah...